pcsx2-qt/Settings/GraphicsSettingsHeader.ui

```diff
@@ -23,7 +23,7 @@
       <item row="0" column="0">
        <widget class="QLabel" name="rendererText">
         <property name="text">
-         <string>Renderer:</string>
+         <string>Graphics API:</string>
```

Comment:
Not needed, same as the other renderer changes.

Reply:
Already been discussed.